components/TextControls/tests/unit/MDCBaseTextFieldTests.m

```diff
@@ -338,4 +338,108 @@ - (void)testPlaceholderVisibility {
                                      labelState:MDCTextControlLabelStateFloating]);
 }
 
+- (void)testDefaultAccessibilityLabelWithOnlyLabelText {
+  // Given
+  CGRect textFieldFrame = CGRectMake(0, 0, 130, 100);
+  MDCBaseTextField *textField = [[MDCBaseTextField alloc] initWithFrame:textFieldFrame];
+  NSString *labelText = @"label text";
+
+  // When
+  textField.label.text = labelText;
```

Comment:
I was expecting textfield's text property to be used.
Why are we using to the label's text?

https://developer.apple.com/documentation/uikit/uitextfield/1619635-text?language=objc

Reply:
The textfield's text property is accounted for by `accessibilityValue` and we're testing `accessibilityLabel` here. I guess we're going to have `accessibilityLabel` be a combination of the label text and the assistive label text and we're going to use the default accessibilityValue behavior.